src/simple/Phone.js

```diff
@@ -67,17 +66,22 @@ class Phone extends Emitter {
       throw new Error('Sorry, no sip lines found for this user');
     }
 
+    this.connectWithCredentials(server, this.sipLine, session.displayName(), options);
+  }
+
+  connectWithCredentials(server: string, sipLine: SipLine, displayName: string, rawOptions: Object = {}) {
```

Comment:
Blind question: if you're creating a new function, i'm guessing you intend to use `connectWithCredentials` directly. Should you consider testing if we're connected already?